lib/tty/prompt/choice.rb

```diff
@@ -67,6 +67,13 @@ def self.convert_hash(val)
       # @api public
       attr_reader :key
 
+      # The text to display when this choice is used with :enum (i.e. in List)
+      # Defaults to :key. Use `nil` or `false` to disable showing the enum for
+      # this Choice.
+      #
+      # @api public
+      attr_reader :key_name
```

Comment:
In light of the previous comment about `:enum` option, this comment would potentially need to change to reflect the changes